bg-find-sync/Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG  DOCKER_REPOSITORY
+ARG  MANAGE_COURSES_BACKEND_IMAGE
+ARG  CODE_VERSION=latest
+
+FROM ${DOCKER_REPOSITORY}/${MANAGE_COURSES_BACKEND_IMAGE}:${CODE_VERSION}
+
+CMD bundle exec rails db:migrate && bundle exec bin/delayed_job --queue=find_sync run
```

Comment:
This duplicates `Dockerfile.bg-find-sync`. We should only have one `Dockerfile` for `bg-find-sync`.

Reply:
removed Dockerfile.bg-find-sync